charts/akeyless-api-gateway/templates/cache.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if not (eq (include "akeyless-api-gw.adminAccessUidExist" .) "")  }}
+{{- if or (not (eq (include "akeyless-api-gw.adminAccessUidExist" .) "")) ( .Values.cachingConf.useClusterCache) }}
```

Comment:
why the "or" condition with "useClusterCache"? should it be "cachingConf.enabled" ?

Reply:
actually for cluster cache we need both:
- `cachingConf.enabled`
- `cachingConf.useClusterCache`